src/lib/y2network/virtualization_config.rb

```diff
@@ -72,14 +70,44 @@ def create
         bridge_builder.save
 
         # Move routes from the port member to the bridge (bsc#903889)
-        Yast::LanItems.move_routes(builder.name, bridge_builder.name)
+        move_routes(builder.name, bridge_builder.name)
       end
 
       true
     end
 
   private
 
+    def yast_config
+      Y2Network::Config.find(:yast)
```

Comment:
Is this really needed?, as the config is an instance variable which is initialized by the constructor.

Reply:
right